usr/src/cmd/zdb/zdb.c

```diff
@@ -4924,12 +4924,18 @@ zdb_embedded_block(char *thing)
 		exit(1);
 	}
 	ASSERT3U(BPE_GET_LSIZE(&bp), <=, SPA_MAXBLOCKSIZE);
+	buf = malloc(SPA_MAXBLOCKSIZE);
+	if (buf == NULL) {
+		(void) printf("out of memory\n");
```

Comment:
Please use `fprintf(stderr, ...)`

Reply:
It looks like printf() is what is used elsewhere in this function for errors.

Reply:
Then that is another bug to fix.
fprintf(stderr) is what is used predominantly in the file and it's obviously a better option for error reporting.

Reply:
I agree that going to stderr would be better.  Feel free to fix both uses in this func while you're here, or to leave it for another PR.